Yosemite/Yosemite/Model/JustInTimeMessage.swift

```diff
@@ -54,6 +70,41 @@ public struct JustInTimeMessage: GeneratedFakeable, GeneratedCopiable, Equatable
                   title: message.content.message,
                   detail: message.content.description,
                   buttonTitle: message.cta.message,
-                  url: message.cta.link)
+                  url: message.cta.link,
+                  backgroundImageUrl: message.assets[ImageAssetKind.background.baseUrlKey],
+                  backgroundImageDarkUrl: message.assets[ImageAssetKind.background.darkUrlKey],
+                  badgeImageUrl: message.assets[ImageAssetKind.badge.baseUrlKey],
+                  badgeImageDarkUrl: message.assets[ImageAssetKind.badge.darkUrlKey])
     }
+
+    enum ImageAssetKind {
```

Comment:
What do you think of putting `ImageAssetKind` inside a private extension? At the moment we're not calling it externally.

Reply:
Yeah. It will be used more widely with the modal work I think but I've made it private in 25941e1